canonical-websites/snapcraft.io
# Use webpack to build releases scripts

Fixes #1226 Fixes #1229 

### QA
- ./run build
- webpack should be installed and used to build releases js
- ./run or demo
- go to releases page of any snap
- everything should work as before
